Find a functionally equivalent code:android.transition.TransitionSet.pause:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide
 */

@@ -1,7 +1,7 @@
 {
-    super.pause();
+    super.pause(sceneRoot);
     int numTransitions = mTransitions.size();
     for (int i = 0; i < numTransitions; ++i) {
-        mTransitions.get(i).pause();
+        mTransitions.get(i).pause(sceneRoot);
     }
 }

